backend/user_management/routers/admin.py

```
@@ -0,0 +1,153 @@
+    return {"detail": "User deleted successfully"}
+
+#Add product management functionality for admin
+class ProductUpdate(BaseModel):
+    product_name: str
+    category: str
+    description: str
+    price: float
+    approved: bool
+
+@router.get("/products")
+async def get_all_products(db: Session = Depends(get_db), current_admin: User = Depends(get_current_admin)):
```

Ruff (0.8.2)

Missing return type annotation for public function `get_all_products`

(ANN201)